CMakeLists.txt

```diff
@@ -10,6 +10,33 @@ configure_file (
     "${PROJECT_SOURCE_DIR}/inc/cmake_config.hpp"
     )
 
+
+# INCLUDE DIRECTORIES
+include_directories (inc)
+include_directories (lib)
```

Comment:
If a `lib` directory is used, I think it would be preferable to move the ODB libraries there.

Reply:
sure